build.gradle

```
@@ -82,7 +82,7 @@ configurations.all {
         force 'commons-cli:commons-cli:1.3.1'
         force 'org.apache.httpcomponents:httpcore:4.4.12'
         force "org.apache.commons:commons-lang3:3.4"
-        force "org.springframework:spring-core:5.3.26"
+        force "org.springframework:spring-core:5.3.27"
```

```suggestion
        force "org.springframework:spring-core:5.3.28"
```

https://spring.io/blog/2023/06/15/spring-framework-5-3-28-and-6-0-10-available-now/